sdks/python/apache_beam/examples/snippets/transforms/aggregation/count.py

```diff
@@ -0,0 +1,89 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+
+def count_globally(test=None):
+  # [START count_globally]
+  import apache_beam as beam
+
+  with beam.Pipeline() as pipeline:
+    total_elements = (
+        pipeline
+        | 'Create produce counts' >> beam.Create([
+            ('🥕', 3),
```

Comment:
Should we use something other than integers for values. It almost invite reader to add up the values even though this is not the intended action of count.

A simple change from 1 -> value1 2 -> value2 etc. might clear the confusion.

Reply:
That's a good point, changed!